src/components/QRCodeSurface/index.js

```diff
@@ -14,7 +14,7 @@ const defaultProps = {
 };
 
 const QRCodeSurface = ({ children, size, style, xmlns }) => (
-  <svg height={size} style={style} width={size} xmlns={xmlns}>
+  <svg id="qrcode" height={size} style={style} width={size} xmlns={xmlns}>
```

Comment:
Why not pass an `id` so anyone can pass any needed id? It would also allow having unique ids of a page once 2 qr codes are used, for example on the same page.